Symfony/CS/Fixer/Symfony/PhpUnitDedicateAssertFixer.php

```diff
@@ -0,0 +1,214 @@
+    {
+        $content = strtolower($tokens[$assertCallIndex]->getContent());
+        if ('asserttrue' === $content) {
+            $isPositive = 1;
+        } elseif ('assertfalse' === $content) {
+            $isPositive = 0;
+        } else {
+            return $assertCallIndex;
+        }
+
+        // test candidate for simple calls like: ([\]+'some fixable call'(...))
+        $assertCallOpen = $tokens->getNextMeaningfulToken($assertCallIndex);
+        if (!$tokens[$assertCallOpen]->equals('(')) {
+            return $assertCallIndex;
+        }
+
+        $testNsToken = false;
+        $test = $tokens->getNextMeaningfulToken($assertCallOpen);
+
+        if (!$tokens[$test]->isGivenKind(array(T_EMPTY, T_STRING))) {
+            if (!$tokens[$test]->isGivenKind(T_NS_SEPARATOR)) {
+                return $test;
+            }
+
+            $testNsToken = $test;
+            $test = $tokens->getNextMeaningfulToken($test);
+            if (!$tokens[$test]->isGivenKind(T_STRING)) {
+                return $test;
+            }
+        }
+
+        $testOpen = $tokens->getNextMeaningfulToken($test);
+        if (!$tokens[$testOpen]->equals('(')) {
+            return $testOpen;
+        }
+
+        $testClose = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $testOpen);
+
+        $assertCallClose = $tokens->getNextMeaningfulToken($testClose);
+        if (!$tokens[$assertCallClose]->equalsAny(array(')', ','))) {
+            return $assertCallClose;
+        }
+
+        return array(
+            $isPositive,
+            $assertCallIndex,
+            $assertCallOpen,
+            $testNsToken,
+            $test,
+            $testOpen,
+            $testClose,
+            $assertCallClose,
+        );
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param array  $assertIndexes
+     *
+     * @return int index up till processed, number of tokens added
+     */
+    private function fixAssert(Tokens $tokens, array $assertIndexes)
+    {
+        list(
+            $isPositive,
+            $assertCallIndex,
+            $assertCallOpen,
+            $testNsToken,
+            $test,
+            $testOpen,
+            $testClose,
+            $assertCallClose
+        ) = $assertIndexes;
+
+        $content = strtolower($tokens[$test]->getContent());
+        if (array_key_exists($content, $this->fixMap)) {
+            if (false !== $this->fixMap[$content][$isPositive]) {
+                $tokens[$assertCallIndex]->setContent($this->fixMap[$content][$isPositive]);
+                $this->removeFunctionCall($tokens, $testNsToken, $test, $testOpen, $testClose);
+            }
+
+            return $assertCallClose;
+        }
+
+        switch ($content) {
+            case 'is_real':
+                $content = 'is_float';
+                // do not break here
+            case 'is_array':
+            case 'is_bool':
+            case 'is_float':
+            case 'is_int':
+            case 'is_string':
```

Comment:
nice list :)
I'll work on it (note to self: [PHPUnit](https://github.com/sebastianbergmann/phpunit/blob/5.3/src/Framework/Constraint/IsType.php#L36]))